apps/nowcasting-app/components/layout.tsx

```diff
@@ -14,9 +14,10 @@ const Layout = ({ children }: ILayout) => {
         <title>Nowcasting App</title>
         <link rel="icon" href="/favicon.ico" />
       </Head>
-      <Navbar />
-      <main>{children}</main>
-      <Footer />
+      <main className="absolute inset-0 flex flex-col	">
```

Comment:
we need linting plugin, remove white spaces

Reply:
ofcourse 👌  ,can you add a story to config eslint or prettier